tutorials/fundamentals/spread.md

```diff
@@ -0,0 +1,36 @@
+JavaScript's `spread` operator, `...`, is a handy way to pass 
+multiple values from an array, object, or anything you can iterate
+through to a function or variable assignment. It shines especially
+when you want to combine two or more arrays. Instead of having to
+use `push`, `splice`, `concat`, and the kitchen sink,  `...` makes
```

Comment:
I'd remove the kitchen sink bit, and just say "The `...` operator offers a more intuitive way to combine multiple arrays than using `push()`, `splice()`, or `concat()`."